mne/io/eeglab/eeglab.py

```diff
@@ -38,9 +42,18 @@ def _check_mat_struct(fname):
     if not check_version('scipy', '0.12'):
         raise RuntimeError('scipy >= 0.12 must be installed for reading EEGLAB'
                            ' files.')
-    from scipy import io
-    mat = io.whosmat(fname, struct_as_record=False,
-                     squeeze_me=True)
+    try:
+        #Try to read old style Matlab file
+        mat = whosmat(fname, struct_as_record=False, squeeze_me=True)
+    except:
```

Comment:
if there is no way of knowing file type before loading and you have to use try-catch - it would be better to catch specific case here.